cms/src/main/java/uk/nhs/digital/apispecs/swagger/IsAnyTrueHelper.java

```diff
@@ -0,0 +1,39 @@
+package uk.nhs.digital.apispecs.swagger;
+
+import com.github.jknack.handlebars.Helper;
+import com.github.jknack.handlebars.Options;
+import org.apache.commons.lang.Validate;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Optional;
+
+public class IsAnyTrueHelper implements Helper<String> {
+
+    public static final IsAnyTrueHelper INSTANCE = new IsAnyTrueHelper();
+
+    public static final String NAME = "isAnyTrue";
+
+    @Override public Object apply(final String items, final Options options) throws IOException {
+
+        Validate.notEmpty(options.params,"One or more input parameters");
+        Validate.noNullElements(options.params,"One or more input parameters");
```

Comment:
Similar [comment to the previous one](https://github.com/NHS-digital-website/hippo/pull/1076/files#diff-07955cf23a101e3697055922bbf7714e72910000b41fa2eaeb94da5af7de09d1R19); as it is now the message is doesn't add much value.